201-rbac-managedidentity-maps/azuredeploy.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "subscriptionId": {
```

Comment:
remove this from the template, it won't deploy in any other subscription than the current one

Reply:
done